docs/user/guides/subprojects.rst

```diff
@@ -1,57 +1,78 @@
-How to manage subprojects
-=========================
+Managing subprojects
+====================
 
-This guide shows you how to manage subprojects,
-which is a way to host multiple projects under a "main project".
+Subprojects can be used to nest projects together under a single parent project,
+which is most commonly used to share a single custom domain across multiple projects.
+
+The subproject **alias** field is used to fine tune the URL structure of your built documentation.
```

Comment:
This feels a bit out of place? Seems like it should go in the UI reference section?

Reply:
Yeah, I wasn't sure on this, the alias field was explained a few different ways previously.

The underlying subproject docs describe it better, I think we can just call this explanation out of scope for a guide doc